plugin/trino-databend/src/main/java/io/trino/plugin/databend/DatabendClientModule.java

```diff
@@ -0,0 +1,76 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.databend;
+
+import com.databend.jdbc.DatabendDriver;
+import com.google.inject.Binder;
+import com.google.inject.Module;
+import com.google.inject.Provides;
+import com.google.inject.Scopes;
+import com.google.inject.Singleton;
+import io.opentelemetry.api.OpenTelemetry;
+import io.trino.plugin.jdbc.BaseJdbcConfig;
+import io.trino.plugin.jdbc.ConnectionFactory;
+import io.trino.plugin.jdbc.DecimalModule;
+import io.trino.plugin.jdbc.DriverConnectionFactory;
+import io.trino.plugin.jdbc.ForBaseJdbc;
+import io.trino.plugin.jdbc.JdbcClient;
+import io.trino.plugin.jdbc.JdbcMetadataConfig;
+import io.trino.plugin.jdbc.credential.CredentialProvider;
+import io.trino.plugin.jdbc.ptf.Query;
+import io.trino.spi.function.table.ConnectorTableFunction;
+
+import java.sql.SQLException;
+import java.util.Properties;
+
+import static com.google.inject.multibindings.Multibinder.newSetBinder;
+import static io.airlift.configuration.ConfigBinder.configBinder;
+import static io.trino.plugin.databend.DatabendClient.DEFAULT_DOMAIN_COMPACTION_THRESHOLD;
+import static io.trino.plugin.jdbc.JdbcModule.bindTablePropertiesProvider;
+
+public class DatabendClientModule
+        implements Module
+{
+    @Override
+    public void configure(Binder binder)
+    {
+        configBinder(binder).bindConfig(DatabendConfig.class);
+        configBinder(binder).bindConfig(DatabendJdbcConfig.class);
+        binder.bind(JdbcClient.class).annotatedWith(ForBaseJdbc.class).to(DatabendClient.class).in(Scopes.SINGLETON);
+        bindTablePropertiesProvider(binder, DatabendTableProperties.class);
+        configBinder(binder).bindConfigDefaults(JdbcMetadataConfig.class, config -> config.setDomainCompactionThreshold(DEFAULT_DOMAIN_COMPACTION_THRESHOLD));
+        binder.install(new DecimalModule());
+        newSetBinder(binder, ConnectorTableFunction.class).addBinding().toProvider(Query.class).in(Scopes.SINGLETON);
+    }
+
+    @Provides
+    @Singleton
+    @ForBaseJdbc
+    public static ConnectionFactory createConnectionFactory(BaseJdbcConfig config, CredentialProvider credentialProvider, DatabendConfig databendConfig, OpenTelemetry openTelemetry)
+            throws SQLException
+    {
+        return DriverConnectionFactory.builder(new DatabendDriver(), config.getConnectionUrl(), credentialProvider).setConnectionProperties(getConnectionProperties(databendConfig)).setOpenTelemetry(openTelemetry).build();
```

Comment:
This line is too long. Please format like this:
```java
        return DriverConnectionFactory.builder(new DatabendDriver(), config.getConnectionUrl(), credentialProvider)
                .setConnectionProperties(getConnectionProperties(databendConfig))
                .setOpenTelemetry(openTelemetry)
                .build();
```

Reply:
Fixed.